test/components/groupTests.ts

```diff
@@ -1,210 +1,208 @@
 ///<reference path="../testReference.ts" />
 
 describe("ComponentGroups", () => {
-  it("append()", () => {
-    let componentGroup = new Plottable.Components.Group();
+  describe("Basic add/remove/has functionality", () => {
```

Comment:
Suggestion for consistency? Always have exactly 3 levels of nesting-ness? In this case 
```
describe component
  describe group
    describe basic functionality
```